migrations/20240823213817_add_feedback.up.sql

```diff
@@ -0,0 +1,27 @@
+-- Add up migration script here
+
+CREATE TYPE feedback_type AS ENUM
+    ('Positive', 'Negative', 'Suggestion', 'Note');
+
+CREATE TABLE mod_feedback
+(
+    id SERIAL PRIMARY KEY NOT NULL,
+    mod_version_id INTEGER NOT NULL,
+    reviewer_id INTEGER NOT NULL,
+    feedback TEXT COLLATE pg_catalog."default" NOT NULL DEFAULT 'No feedback provided.'::text,
+    decision BOOLEAN NOT NULL DEFAULT false,
```

Comment:
i'm not 100% sure if the decision field is necessary now, could it just be two enum values? 

mostly just thinking about what happens in the event decision: true + type: note/suggestion... (and in the ui it'd very likely just look like it was part of the enum)

Reply:
decision is only set through one method, and that can only provide positive/negative

Reply:
this is why i feel like it's redundant, it's tied to the status enum in a way that just makes it feel like another way to represent status. it's one of those "i'll think about it some more" things ig but definitely wouldn't call it resolved yet :sob:

could help if i knew how you expect this to look in the cli/site ui, i'm thinking of something closer to github reviews (like this thing) but it seems like you're expecting something different

Reply:
i'll draw up some concept art later

Reply:
![feedback](https://github.com/user-attachments/assets/79f8e9c9-0d23-4934-8930-22b42b08a5e4)
this is what i'm imagining. the feedback blocks would just keep going down the page.

Reply:
github ate my reply but i was going to say i think i understand what you mean now, so i'm marking this as resolved